content/lua/concepts/coroutines/terms/resume/resume.md

```diff
@@ -0,0 +1,76 @@
+---
+Title: 'resume()'
+Description: 'Resumes a coroutine.'
+Subjects:
+  - 'Computer Science'
+Tags:
+  - 'Functions'
+CatalogContent:
+  - 'learn-lua'
+  - 'paths/computer-science'
+---
+
+The **.resume()** function in Lua resumes a coroutine after it has been paused.
+
+## Syntax
+
+The `.resume()` function resumes a coroutine named `routine`.
+
+```pseudo
+coroutine.resume(routine)
+```
+
+## Example
+
+Consider the following code:
+
+```lua
+local routineX = coroutine.create(
+    function ()
+        for i = 1, 5, 1 do
+            print("Run coroutineX!")
+            if i == 2 then 
+                coroutine.yield() 
+            end
+        end
+    end
+)
+
+local routineY = coroutine.create(
+    function () 
+        for i = 1, 3, 1 do
+            print("Run coroutineY!")
+        end
+    end
+)
+
+coroutine.resume(routineX)
```

Comment:
perhaps the output could be more intuitive if the "coroutine resumed" statements are executed before the function call